pkg/sql/crdb_internal.go

```diff
@@ -1188,15 +1185,6 @@ CREATE TABLE crdb_internal.jobs (
 	},
 }
 
-var enablePerJobDetailedAuthLookups = settings.RegisterBoolSetting(
```

Comment:
don't you need to add this to retiredSettings?

Reply:
Sure. This was never public or in any runbooks so I doubt there are many / any clusters out there using it, but I guess it's good to add it all the same.

Done.